blog for firstweek of engi (#67)
scribd/scribd.github.io: _posts/2020-08-26-engineering-firstweek.md

---
layout: post

> title: "My first week at Scribd engineering"

Reviewer: ```suggestion
title: "My first week in Scribd engineering"
```
Missed this awkward preposition. Going to merge it

tags:
- featured
team: Core Infrastructure

> author: zherner

Reviewer: ```suggestion
author: jasond
```

Just noticed this too, for future reference the author key should be what's on line 97  from authors.yml

---
layout: post
title: "My first week at Scribd engineering"
tags:
- scribd
team: Core Infrastructure
author: zherner
---

> My first week at Scribd engineering!

Author: Removed thanks!

Hello everyone, I'm new at Scribd and here's what the first week is like on the Core Infrastructure engineering team. It's full of smart people, great culture, and cool tech and I wanted to share my experience.

> The journey starts of course, with an interview phase. This round has to be one of the most pleasant interview experiences I've had. I've been through the wringer before. From start to finish I was clear on next steps and what to expect. The recruiter was awesome, she scheduled prep meetings with me to answer questions before interviews. I got to meet my manager, the team director, all but one of my teammates (he was on PTO) and folks on other teams, even teammates in Amsterdam. The process was entirely remote due to the pandemic but that didn't take away from the experience. To me it felt just as personal as an onsite would have been by making excellent use of Google meet.

Reviewer: ```suggestion
The journey starts of course, with an interview phase. I've been through the wringer before but this round has to be one of the most pleasant interview experiences I have had. From start to finish I was clear on next steps and what to expect. The recruiter was awesome, she scheduled prep meetings with me to answer questions before interviews. I got to meet my manager, the team director, all but one of my teammates, who was on PTO, and folks from other teams, even teammates in Amsterdam. The process was entirely remote due to the pandemic but that didn't take away from the experience. To me it felt just as personal as an onsite would have been by making excellent use of Google Meet.
```

Suggest a slight reordering to have a cleaner flow

> The journey starts of course, with an interview phase. This round has to be one of the most pleasant interview experiences I've had ….I've been through the wringer before. From start to finish I was clear on next steps and what to expect. The recruiter was awesome, she scheduled prep meetings with me to answer questions before interviews. I got to meet my manager, the team director, all but one of my teammates (he was on PTO) and folks on other teams, even teammates in Amsterdam. The process was entirely remote due to the pandemic but that didn't take away from the experience. To me it felt just as personal as an onsite would have been by making excellent use of Google meet.

Reviewer 2: I'd drop the ` …` </nitpick>

Author: Removed. Thanks!

> Thankfully I'm not alone at first day orientation, I'm joined by a group of new hires. Since we are all in the same boat we've started our own new-hire slack channel to support each other. To get started there is a helpful and thorough checklist for us newbies to walkthrough for setting up the basics. The better part is the team specific onboarding guide that is focused on setup tasks that pertain to tools I'll use for everyday type work. The best part though, has got to be that I got a new buddy! Scribd does new-hire buddies to show you the ropes. It goes a long way to take a little bit of the new-job tension off. You can go to your new hire buddy for anything from onboarding to team specific questions. In my case my buddy is a tech lead so he has a wealth of knowledge on our stack or knows who to point you to if it isn't in their domain.

Reviewer: ```suggestion
When joining a new company, there is always tons to learn and it can be a little overwhelming.Thankfully I was not alone at my first day orientation, I was joined by a group of new hires. Since we were all in the same boat, we started our own new-hire slack channel to support each other. To get started there was a helpful and thorough checklist for newbies to walk through for setting up the basics of our benefits, systems access, and other necessities. The team specific on-boarding guide focused on setup tasks that pertain to tools I would use for everyday work. The best part though, has got to be that I got a new buddy! Scribd pairs new-hires with buddies to show you the ropes. It goes a long way to alleviate a bit of that new-job tension. You can go to your new-hire buddy for anything from on-boarding to team specific questions. In my case my buddy is a tech lead so he also had a wealth of knowledge on our stack or knows who to point you to for answers.
```

Transition between paragraphs was a little abrupt, tried to smooth it over.

Also tweaked to use past tense similar to the previous paragraph

If there were any changes I made which don't capture your voice, feel free to pick and choose.

> Thankfully I'm not alone at first day orientation, I'm joined by a group of new hires. Since we are all in the same boat we've started our own new-hire slack channel to support each other. To get started there is a helpful and thorough checklist for us newbies to walkthrough for setting up the basics. The better part is the team specific onboarding guide that is focused on setup tasks that pertain to tools I'll use for everyday type work. The best part though, has got to be that I got a new buddy! Scribd does new-hire buddies to show you the ropes. It goes a long way to take a little bit of the new-job tension off. You can go to your new hire buddy for anything from onboarding to team specific questions. In my case my buddy is a tech lead so they have a wealth of knowledge on our stack or knows who to point you to if it isn't in their domain.

Reviewer 2: `my buddy is a tech lead so they have a` "he/she" feels more natural here than "they", but up to you 😀

Author: Agreed. Got it in.

I was happy to hear that Scribd does new hire meet and greets to introduce managers and key people from other teams to start building collaborative relationships. Typically they are from teams you'd work closely with so that you can add more faces to your cache and get to know people. This is huge in getting comfortable with the team's area of responsibilities and who knows what. Everyone I've met at Scribd is crazy smart but more importantly super friendly which makes growing into the new job much smoother and easier. There are fun Slack channels for all kinds of things like coffee, cooking, pets, sports, cars, you name it making it easy to nerd out and meet new people.

> As for the tech stack, some of it is completely new to me but that makes it exciting to learn new things and get different perspectives on solving problems. It also makes for many learning opportunities from the folks who are SMEs on areas of the stack. Being on the core infrastructure team I get to keep up with Terraform, AWS, Kubernetes and fresh new tech that comes. Our team builds and maintains the infrastructure foundation that is shared by the other engineering teams. For me the cool thing is being part of many engineering efforts and getting to share and learn knowledge from each team and each project.

Reviewer: ```suggestion
As for the tech stack, some of it was completely new to me, but that will make it exciting to learn all new things and get different perspectives on solving problems. It also makes for many learning opportunities from the folks who are subject-matter experts on areas of the stack. Core Infrastructure builds and maintains the foundation that is shared by many other engineering teams. This means that I get to keep up with Terraform, AWS, Kubernetes and more fresh new tech that becomes available in the infrastructure space. For me the cool thing is being part of many engineering efforts and getting to share and learn knowledge from each team and each project.
```

Made a little more past tense, re-ordered some sentences to flow a little cleaner

> As for the tech stack, some of it is completely new to me but that makes it exciting to learn new things and get different perspectives on solving problems. It also makes for many learning opportunities from the folks who are SMEs on areas of the stack. Being on the core infrastructure team I get to keep up with terraform, AWS, kubernetes and fresh new tech that comes. Our team builds and maintains the infrastructure foundation that is shared by the other engineering teams. For me the cool thing is being part of many engineering efforts and getting to share and learn knowledge from each team and each project.

Reviewer 2: I'd capitalize Terraform and Kubernetes :-)

Author: T, K. thanks!

> My first day had other exciting news too! Scribd announced the acquisition of Slideshare, which, in current times it is great to see Scribd is doing well and making progress even as so much of the world faces trying challenges.

Reviewer: ```suggestion
Any first week at Scribd is going to be memorable but I was especially lucky in that my first day [Scribd announced the acquisition of SlideShare](https://blog.scribd.com/home/welcome-slideshare-to-the-scribd-community). I can't guarantee that your first day will be _that_ exciting, but I'm certain [your first week will be a good one](https://tech.scribd.com/careers/#open-positions).
```

The end of the article was a little abrupt, so I tried to ease the reader out a little more.

The "challenges" line was a little bit of a bummer, so I cut it to try to end a little more upbeat :smile:
